ci/tox/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN set -x \
     && cd git-$git_version/ \
     && make configure \
     && ./configure \
+    && sed -i 's/#warning .*//' /usr/include/sys/poll.h  `# to reduce amount of spam warnings in logs` \
```

Comment:
This change, while useful on its own, will have a side effect of a new `gitmachete/ci-tox` Docker image being generated (since the SHA-1 git hash of `ci/tox/` directory changes)... which serves as kinda-workaround for #396 (the new image will use the UID/GID of the VM user), but only until CircleCI provides a VM with a new UID/GID